core/internal/gethwrappers/generation/generate.go

```diff
@@ -0,0 +1,9 @@
+///bin/true; exec /usr/bin/env go run "$0" "$@"
```

Comment:
This is awesome.

Reply:
Oh, damn, I didn't mean to check that in. Still, it's cool, and means we can run go with `go generate` commands.